index.js

```diff
@@ -75,14 +75,26 @@ function buildContent(github, owner, repo, base, head, file) {
   }
 }
 
-function getContent(github, owner, repo, path, commit) {
+function getContent(github, owner, repo, path, commit, sha) {
   return github.repos.getContent({
     owner,
     repo,
     path,
     ref: commit,
   }).then((res) => {
     return res.content;
+  }).catch((err) => {
+    try {
+      const apiError = JSON.parse(err);
+      if (apiError.errors.find(error => error.code === 'too_large')) {
+        return github.gitdata.getBlob({
+          owner,
+          repo,
+          sha
+        }).then(data => data.content);
```

Comment:
It looks like blob content is always returned as base64 encoded: https://developer.github.com/v3/git/blobs/#get-a-blob

Shouldn't we be un-encoding it here so that the diff can work properly?

Reply:
`getContent` also returns a base64 encoded content. The `atob` function is used in both cases to un-encode the content.

Reply:
good catch - it's obviously been a while since I worked on this. the repo could really use some tests - in fact if you have some time... :stuck_out_tongue: